.github/workflows/tests.yml

```diff
@@ -13,8 +13,11 @@ jobs:
         with:
           python-version: 3.8
       - name: Install dependencies needed to download files
+        # we're just installing mpol here to reference the zenodo record number 
```

Comment:
Can you explain why the zenodo reference number is needed at this point in the tests? Is it for the later `download_external_files.py` stage? I think this is a fine solution, but if you really wanted to defer installation of the package until the next stage of the workflow, you could modify the download files to take a path to a zenodo reference, and you would have a script that reads that path from the __init__.py source file (something similar to the [single source Python version](https://packaging.python.org/en/latest/guides/single-sourcing-package-version/) but this is probably overkill).

Reply:
Yes that's right, the reference number is needed for `download_external_files.py` to reference. It's a little clunky, but yeah I think it's fine. Your reference is a good one, but if you're ok with a bit of redundancy at this point in the workflow, I'm happy to keep the current workaround.